### PR TITLE
Logging for snare and cloner

### PR DESCRIPTION
**Pull request by viskey98**
fixes issue #73 
I have created a logger,
which stores debug and info logs into : "snare.log"
and error logs into : "snare.err"
Please suggest any changes if required :)

### CODE REVIEW

**Comment by mzfr** [comment]
Hey! I already had a logging PR under review. :disappointed:

**Comment by viskey98** [comment]
@mzfr I thought of creating a PR with many improvements and including error log as well. Also the PR (as it is a feature implementation) is required to be done on develop branch. So i created a fresh one on my own.

**Comment by viskey98** [comment]
I have added a error logger for cloner side as well.
Fixes issue #110 
The error logs for cloner will now be logged in "cloner.err"

File: snare.py [hunk 1]
```diff
@@ -453,12 +457,20 @@ def parse_timeout(timeout):
...
+    parser.add_argument("--log_path", help="path to the log file")
     args = parser.parse_args()
     base_path = '/opt/snare/'
     base_page_path = '/opt/snare/pages/'
     config = configparser.ConfigParser()
     config.read(os.path.join(base_path,args.config))
-
+	
+    if args.log_path:
+        log_debug = args.log_path + "snare.log"
```
Comment (afeena) [inline snare.py]: it is not clear that `log_path` is a directory. If I were new for snare, I would put full path to log something like /foo/bar/snare.log

File: snare.py [hunk 2]
```diff
@@ -126,11 +130,11 @@ def create_data(self, request, response_status):
                     try:
                         event_result = await r.json()
                     except json.decoder.JSONDecodeError as e:
-                        print(e, data)
+                        self.logger.error('Error submitting data: {} {}'.format(e, data))
                     finally:
                         r.release()
         except Exception as e:
-            raise e
```
Comment (afeena) [inline snare.py]: raise is not equivalent to print, please keep it here :)

**Comment by viskey98** [comment]
@afeena Requested changes have been made, please look into it :)

**Comment by afeena** [comment]
With help of #81 (Thanks to @mzfr)

**Pull request merged**